aten/src/ATen/native/native_functions.yaml

```diff
@@ -1244,6 +1244,8 @@
     CPU: flip_cpu
     CUDA: flip_cuda
 
+- func: rot90(Tensor self, int64_t k, IntList dims) -> Tensor
```

Comment:
btw `dims` should be `IntList[2]` and have default value `{0, 1}`. Not sure if default values work on IntList, but could try it and make sure everything works (including python arg parser etc)?

Reply:
Ah, ok. I will add default values `k=1` and `dims={0, 1}`. I was trying to add them but wasn't quite sure if `dims={0, 1}` is intuitive enough for n-D tensors.

Reply:
NumPy does it so I think it's fine. :)

Reply:
Sounds like IntList does not support default values in format of `{0, 1}` as [native_parse](https://github.com/pytorch/pytorch/blob/5642937ac193168aad249c54a5533e5eb8fbb909/aten/src/ATen/native_parse.py#L11-L25) does not capture this case. Do you know if it is possible to add this pattern to the parser?

Reply:
You might need to tweak codegen a bit, but it should be possible.